src/test/lib/file-utils.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const fs = require('fs');
+const matcher = require('matcher');
+
+const listFilesInDirectory = (dir, excludes = []) => {
```

Comment:
I am not fan of this code here, as this code has some logic, not sure if it should belong in the test repo as well, we know the folder structure before the hand, wouldn't it be better if we hardcode the expected test output immediately instead of relying on this? Just to simplify the tests.

Reply:
Can do just, left it there as a left over

Reply:
Have removed and hardcoded a list in `json2xlsx`